lesson02/elb_asg/data_sources.tf

```diff
@@ -0,0 +1,43 @@
+# data "aws_vpc" "selected" {
```

Comment:
remove this if it is not being used.

Reply:
o ya vi, ya vi: `yes, let me update it, I will leave a comment on how to use the DATA resource just to explain it during the session`